src/libraries/AccountAccessParser.sol

```diff
@@ -201,6 +201,46 @@ library AccountAccessParser {
     }
 
     /// @notice Extracts all unique storage writes (i.e. writes where the value has actually changed)
+    function getUniqueWritesNew(VmSafe.AccountAccess[] memory accesses)
```

Comment:
The diff between `getUniqueWritesNew` and `getUniqueWrites` if small. I'm including the fix as a separate function until we confirm the issue. Once we confirm it I will swap out `getUniqueWrites` for `getUniqueWritesNew`.

Reply:
This [commit](https://github.com/ethereum-optimism/superchain-ops/pull/656/commits/be380d3a9dd112215084c52456003c8c765f4226) shows the _actual_ diff more clearly.

Reply:
Thanks closing.